src/main/java/com/internet/shop/controllers/order/GetAllOrdersOfUserAdminController.java

```diff
@@ -0,0 +1,26 @@
+package com.internet.shop.controllers.order;
+
+import com.internet.shop.lib.Injector;
+import com.internet.shop.model.Order;
+import com.internet.shop.service.interfaces.OrderService;
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class GetAllOrdersOfUserAdminController extends HttpServlet {
+    public static final Long USER_ID = 1L;
+    private static final Injector injector = Injector.getInstance("com.internet.shop");
+    private OrderService orderService =
+            (OrderService) injector.getInstance(OrderService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        List<Order> orders = orderService.getUserOrders(USER_ID);
```

Comment:
```suggestion
        List<Order> orders = orderService.getAll();
```

Reply:
ok, I did it for a specific user a specially but fixed it to all